client/src/App.js

```diff
@@ -22,7 +22,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList}/>
```

Comment:
Can also move less specific routes to the bottom to get the desired effect from exact (though exact is still a good idea with home pages)